epub33/rs/index.html

```diff
@@ -2476,9 +2476,13 @@ <h3>Recommendations</h3>
 					the validation steps that usually occur. This could include running virus scans, validating external
 					links and remote resources, and other precautions.</p>
 
-				<p id="security-privacy-recommendations-sideloaded-consent" data-tests="#sec-untrusted-consent_network,#sec-untrusted-consent_scripting">Reading systems that allow users to load untrustworthy EPUB publications (e.g., unsigned EPUB
-					publications through the process of "sideloading") SHOULD treat such content as insecure (e.g.,
-					prompt users to allow scripting and network access).</p>
+				<p id="security-privacy-recommendations-sideloaded-consent" 
+					data-tests="#sec-untrusted-consent_network,#sec-untrusted-consent_scripting">Reading systems 
+					SHOULD treat content from any new source, including  from an integrated bookstore, as insecure 
```

Comment:
two spaces after including should be one

Reply:
Yep, done